packaging/language/bower.py

```diff
@@ -172,13 +193,14 @@ def main():
     offline = module.params['offline']
     production = module.params['production']
     path = os.path.expanduser(module.params['path'])
+    relative_execpath = os.path.expanduser(module.params['relative_execpath'])
```

Comment:
I believe this will fail when relative_execpath is not specified:

``` python
>>> os.path.expanduser(False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/posixpath.py", line 261, in expanduser
    if not path.startswith('~'):
AttributeError: 'bool' object has no attribute 'startswith'
```